src/main.go

```diff
@@ -246,7 +332,7 @@ func process(data []models.DmwKnownContact, dfoAuthTokenObj models.DfoAuthTokenO
 		// Push sanitizedUpdateRecords to digimiddleware via gRPC
 		if sanitizedUpdateRecords != nil {
 			method2 := "sendUpdateRecordsToMiddleware"
```

Comment:
Could also be moved outside of the loop.

Naming something method2 isn't exactly descriptive.  Rename to something like methodUpdateRecords.

Could potentially be a set of "method" const values at root level?

Reply:
I will move them outside of the loop.  For now, I am going to keep them as method and method2 because I feel that this will likely change and it will help me to find them to refactor them all.